dipy/segment/tests/test_quickbundles.py

```diff
@@ -72,7 +77,12 @@ def test_quickbundles_2D():
     data += [rng.randn(1, 2) + np.array([-10, -10]) for i in range(5)]
     data = np.array(data, dtype=dtype)
 
-    clusters_truth = [[0], [1, 2], [3, 4, 5], [6, 7, 8, 9], [10, 11, 12, 13, 14]]
+    clusters_truth = [
+        [0], [
+            1, 2], [
+            3, 4, 5], [
+                6, 7, 8, 9], [
+                    10, 11, 12, 13, 14]]
```

Comment:
But with the indentation of the second line onwards correctly matched (it's a bit hard to do with the comment text boxes on Github :-D)